### PR TITLE
Fix NPE during building docker image when image with null tag exists

### PR DESCRIPTION
**Pull request by vasilevichra**
I haven't forgotten about:
- [ ] Tests
- [x] Changelog
- [ ] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:

### CODE REVIEW

File: src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java [hunk 1]
```diff
@@ -71,6 +72,7 @@ private static boolean hasImage(String tag) {
         final List<Image> images = getDockerClient().listImagesCmd().exec();
         return images.stream()
                 .map(Image::getRepoTags)
+                .filter(Objects::nonNull)
```
Comment (ArtDu) [inline src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java:75]: Should we move this below right before `.anyMatch`?
I thought that exactly repoTag was null

Comment (vasilevichra) [reply src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java:75]: No, we can't. In this case array of tags is null.

**Review by ArtDu — COMMENTED** [review]
Change 3.0.1 to 3.0.1-old in CI file

File: src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java (hunk 1 above)
Comment (ArtDu) [reply src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java:75]: I believe you that you have tested and got success result

**Review by ArtDu — CHANGES_REQUESTED** [review]
Try to follow commit [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)

First of all:
* 50 symbols in commit title 
* Do not end the subject line with a period


image

If you need to expand commit you can type many lines in commit body

**Review by ArtDu — APPROVED** [review]

**Review by ArtDu — APPROVED** [review]

**Pull request merged**